examples/resources/kentik-synthetics_test/resource.tf

```diff
@@ -78,3 +78,144 @@ resource "kentik-synthetics_test" "example-hostname-test" {
 output "hostname-test" {
   value = kentik-synthetics_test.example-hostname-test
 }
+
+resource "kentik-synthetics_test" "minimal-hostname-test" {
+  name      = "minimal-hostname-test"
+  type      = "hostname"
+  status    = "TEST_STATUS_PAUSED"
+  settings {
+    hostname {
+      target = "www.example.com"
+    }
+    agent_ids = [
+      "817",
+      "818",
+      "819"
+    ]
+    tasks = [
+      "ping",
+      "traceroute"
+    ]
+    monitoring_settings {
+      activation_time_unit    = "m"
+      activation_time_window  = "5"
+      activation_times        = "3"
+    }
+    ping {
+      period = 60
```

Comment:
`settings.protocol` is missing. Does the provider or the SDK default it to something meaningful? It is used by the `ping` task.

Reply:
Last week settings.protocol was optional and the backend set it to "tcp" by default. Now it is required, so I marked it as required in schema and added values to examples.

Reply:
Thanks. The automatic setting to TCP was actually a bug in the API, which caused problems with tests against public agent, which are not reachable via arbitrary TCP ports.